### ISSUE TITLE
Cant' install chrome web store extension - Fails with access denied after downloading .crx file

### ISSUE THREAD

**Issue by moritzhaller** [brave/brave-browser#6753]
[x] Have you searched for similar issues? Before submitting this issue, please check the open issues and add a note before logging a new issue.

Found related issue in Brave community: https://community.brave.com/t/cant-install-lastpass-extension-in-brave/72115

## Description 

- I have developed a custom browser extension for ML user studies that should be distributed through the chrome web store (for whitelisted users only for now)
- The extension installs as expected in Chrome but fails to install in Brave
- When clicking the "Add to Brave" button, the installation fails with the error "An error has occurred -Access denied" after downloading a `.crx` file (see screenshot below)
- The same occurs for a "debug" extension that asks for minimal rights in its `manifest.json`

## Steps to Reproduce

   1. Open Brave
   2. Ask @moritzhaller to whitelist you as test user for the extension 
   3. Visit [extension website in chrome web store](https://chrome.google.com/webstore/detail/brave-data-collector-debu/jpgjincahmolliagodcdaefeamccbinp)

## Actual result:

<img width="1435" alt="Screenshot 2019-11-04 at 14 11 33" src="https://user-images.githubusercontent.com/1259879/68128983-aab41900-ff10-11e9-8be1-d8728b3a8c5c.png">

## Expected result:

Successfully installed extension

## Reproduces how often:

Easily reproduced on two personal laptops (both Macbook pro/Mojave)

## Brave version (brave://version info)

Brave | 0.70.122 Chromium: 78.0.3904.87 (Official Build) (64-bit)
Revision | 20c21f4010010f32462ea8e1d6af30cef66d48c8-refs/branch-heads/3904@{#840}
OS | macOS Version 10.14.6 (Build 18G95)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Didn't test
- Can you reproduce this issue with the dev channel? Didn't test
- Can you reproduce this issue with the nightly channel? Didn't test

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by rebron** [comment]
blocked on #7167

**Comment by jantimon** [comment]
I have the same behaviour even if I am not using private mode.

Brave Version 1.2.42 Chromium: 79.0.3945.117 (Official Build) (64-bit)

**Comment by flowli** [comment]
@jantimon Same behaviour here with exact same Brave/Chromium on macOS.

**Comment by seanbebliss** [comment]
Same behavior. Not private mode. Shields down. 
Under shield button: Allowed all cookies on this site, allowed all finger printing on this site, allowed all trackers on this site.  Still getting the error. 
Version 1.25.70 Chromium: 91.0.4472.77 (Official Build) (64-bit) (p.s. - NEVER UPDATE AGAIN INVASIVE CIRCLE TRIANGLE ICON HATE SEEING IT MAKE IT GO AWAY! ROLL BACK TOO TECHNICAL (where easy install old version? NONE!!!!). YOU BREAKING MY MIND STUPED ALWAYS VISIBLE BUTTON!)

**Comment by jquintanilla4** [comment]
Same behavior. Not private. Shields down.
Shield settings:
Trackers & ads blocking - Aggressive 
Upgrade connections to HTTPS - ON
Block Scripts - OFF
Cookie blocking - Only cross-site
Fingerprinting blocking - Standard

Turned off brave shield on Chrome Web Store.
Version 1.26.67 Chromium: 91.0.4472.114 (Official Build) (x86_64)
MacOS Big Sur 11.4 (20F71)

**Comment by DeadZombie14** [comment]
Same behavior here. Not private. Shields down. Google login on extensions enabled. All cookies allowed. Brave version 1.26.67.

**Comment by seanbebliss** [comment]
**UPDATE:** Problem Solved! ?
It's been 2 days. I shutdown the computer. Today I clicked extensions to see the ones I already have. There there is "developer options" switch, which was already on from last time, on it's horizontal bar at the top is "update" button. Clicked it, current extension successfully updated! Whaaaaa? Tried the webstore again, all extensions successfully downloaded and installed on this attempt. 

Solutions? 1. Wait, 
2. Restart P.C. (maybe it's a Windows file lock like those "can't delete file" problems), or 
3. click the extensions' "update" button under extension developer options.  

I'm guessing it's 2: P.C. restart that did it because "access denied" sounds like a file won't move or delete Microsoft problem. That's usually a Windows restart issue.

**Comment by seanbebliss** [comment]
**NOPE:**

Brave says it was their fault. They fixed it. #16611  https://github.com/brave/brave-browser/issues/16611

**Comment by bsclifton** [comment]
@seanbebliss we'd need to implement https://github.com/brave/brave-browser/issues/7167 in order for this issue to be resolved. You might want to follow that one for updates

The issue we're commenting in covers situation where you get an error trying to access a restricted extension (ex: Google cookie sent to authenticate you are in allow list for extension)

The issue you linked to above (https://github.com/brave/brave-browser/issues/16611) was specifically due to a backend outage we had on our extension update server
